components/SwipeListView.js

```diff
@@ -26,6 +26,11 @@ class SwipeListView extends Component {
             };
         }
     }
+    
+    componentWillReceiveProps(nextProps) {
+		if(nextProps.refreshing)
```

Comment:
Could you make sure this has been run through prettier and eslint? The spacing looks off here and please add curly braces

Reply:
Oh yeah, looks like there was something up and my linter wasn't running.

Reply:
Should be all set now.